pkg/machine/keys.go

```diff
@@ -51,7 +52,22 @@ func CreateSSHKeysPrefix(dir string, file string, passThru bool, skipExisting bo
 // generatekeys creates an ed25519 set of keys
 func generatekeys(writeLocation string) error {
 	args := append(append([]string{}, sshCommand[1:]...), writeLocation)
-	return exec.Command(sshCommand[0], args...).Run()
+	cmd := exec.Command(sshCommand[0], args...)
+	stdErr, err := cmd.StderrPipe()
```

Comment:
I think a better way to do this is: 

```
cmd.Stdout = os.Stdout
cmd.Stderr = os.Stderr
cmd.Stdin = os.Stdin
```

At least that's how I always exec my commands! In this case too you do not need to worry about catching and returning the error I don't think

Reply:
This will always print the output even on success (depends if the command prints something).
I think the idea is to only provide the error message in case it didn't work, so I think this is fine.

Reply:
Concur, pipes seem appropriate since we only really want to log on error